LrcParser/Parser/Lrc/Utils/LrcStartTimeUtils.cs

```diff
@@ -8,6 +8,19 @@ namespace LrcParser.Parser.Lrc.Utils;
 
 public static class LrcStartTimeUtils
 {
+    /// <summary>
+    /// Check if the line starts with a line time tag.
+    /// </summary>
+    /// <param name="line">The lyrics line in the LRC format.</param>
+    /// <returns>
+    /// <c>true</c> if the line starts with a line time tag, <c>false</c> otherwise.
+    /// </returns>
+    internal static bool StartsWithLineTimeTag(string line)
```

Comment:
If the previous comment is right, this is not needed also.

Reply:
Do we maybe want to keep this anyway and expose it to the API surface? Might be useful for users of the library. But we can also remove it again, your decision.

Reply:
OK to keep this.